internal/rbd/nodeserver.go

```diff
@@ -149,6 +149,65 @@ func healerStageTransaction(ctx context.Context, cr *util.Credentials, volOps *r
 	return nil
 }
 
+// populateVolOptions update the fields in rbdVolume struct based on the request it received.
+func (ns *NodeServer) populateVolOptions(
+	ctx context.Context,
+	req *csi.NodeStageVolumeRequest,
+	disableInUseChecks bool,
+	isStaticVol bool,
+	volID string, cr *util.Credentials) (*rbdVolume, error) {
+	var err error
+	volOptions, err := genVolFromVolumeOptions(ctx, req.GetVolumeContext(), req.GetSecrets(), disableInUseChecks)
```

Comment:
`volOptions` is not really a suitable name better call it `rbdVol` or `rv` like done in other places.

Reply:
changed and kept it with rbdvol

Reply:
it has not changed, still `volOptions` and not `rbdVol`?

Reply:
@nixpanic can you refresh and see ?  it is `rbdVol` https://github.com/ceph/ceph-csi/pull/2482/files#diff-2f84daaad04728739d84bfee31afc54913845fde94c8d612079dc36f45bdb5f6R153

Reply:
On line 181 the variable is still confusingly called `volOptions`, not `rv` or `rbdVol`. Refreshing does not seem to be sufficient.

Reply:
changed those too..